npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -74,6 +74,10 @@ def updated_lockfile_reponse(response)
         SOCKET_HANG_UP = /request to (?<url>.*) failed, reason: socket hang up/
         UNABLE_TO_AUTH_NPMRC = /Unable to authenticate, need: Basic, Bearer/
         UNABLE_TO_AUTH_REGISTRY = /Unable to authenticate, need: *.*(Basic|BASIC) *.*realm="(?<url>.*)"/
+        MISSING_AUTH_TOKEN = /401 Unauthorized - GET (?<url>.*) - authentication token not provided/
+        INVALID_AUTH_TOKEN =
+          /401 Unauthorized - GET (?<url>.*) - unauthenticated: User cannot be authenticated with the token provided./
+        GIT_PACKAGE = "https://npm.pkg.github.com"
```

Comment:
is this only applicable to npm?

Reply:
If so, should we rename to `NPM PACKAGE REGISTRY` or something like that?

Reply:
fixed